synapse/lib/stormlib/vault.py

```diff
@@ -0,0 +1,738 @@
+import synapse.exc as s_exc
+import synapse.common as s_common
+
+import synapse.lib.cell as s_cell
+import synapse.lib.stormtypes as s_stormtypes
+
+stormcmds = (
+    {
+        'name': 'vault.add',
+        'descr': '''
+            Add a vault.
+
+            Examples:
```

Comment:
The examples are using --scope which is not in cmdargs; cruft?

Reply:
Yep, good catch, thanks!